### DISCUSSION TITLE
Migrating to new host

### DISCUSSION THREAD

**Discussion by DeltaTango69** [immich-app/immich#12575] [Q&A]
Hello there,

I want to migrate from my Windows with WSL2 to a new Mini-PC with Debian.
I will kept to this description : https://immich.app/docs/administration/backup-and-restore/

But I wonder what is with the thumbs folder. Can I copy this and all thumbs are recognized again? I have tested this on my old host, but something went wrong the thumbs were not recognized again?

**Comment by bo0tzz** [comment]
If you move everything over exactly then it should pick up the thumbs as well. If it doesn't that indicates something went wrong. Another option is to not move the thumbs and regenerate them on the new system.

**Reply by DeltaTango69** [reply]
It worked fine . I have copied all and not only the thumbs. Problem was the dump.sql from windows and for restore in Linux I need the dump.sql.gz ?!
